actors/builtin/miner/monies_test.go

```diff
@@ -73,4 +73,38 @@ func TestPledgePenaltyForTermination(t *testing.T) {
 				bigInitialPledgeFactor))
 		assert.Equal(t, expectedFee, fee)
 	})
+
+	t.Run("fee for replacement = fee for original sector when power, BR are unchanged", func(t *testing.T) {
```

Comment:
Nice to have -- a case showing that past the cutoff time the termination fee does not depend on old sector age or old sector br.